charts/prefect-server/values.schema.json

```diff
@@ -715,7 +715,209 @@
     "global": {
       "type": "object",
       "title": "Global",
-      "description": "global configuration.  Not set by user but required when prefect is referenced as a downstream Chart"
```

Comment:
@parkedwards @mitchnielsen lmk if this will be problematic

Reply:
this description change? it's fine w me

Reply:
nah this was the the use of global for the prefect stuff -- i've already addressed!